sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorReceiver.java

```diff
@@ -71,16 +72,11 @@ public class ReactorReceiver implements AmqpReceiveLink, AsyncCloseable, AutoClo
     private final AtomicLong lastSequenceNumber = new AtomicLong();
     private final AutoCloseable trackPrefetchSeqNoSubscription;
 
-    @Deprecated
+    // Note: ReceiveLinkHandler2 will become the ReceiveLinkHandler once the side by side support for v1 and v2 stack
+    // is removed. At that point the type "ReceiveLinkHandlerWrapper" type will be removed and the Ctr will take
+    // "ReceiveLinkHandler".
     protected ReactorReceiver(AmqpConnection amqpConnection, String entityPath, Receiver receiver,
```

Comment:
I would promote this to a javadoc so when people hover over it, they know its intention.

Reply:
Make sense. I’m thinking we can move this comment as the javadoc for the type `ReceiveLinkHandlerWrapper`, so it's in one place instead of repeating it in places wrapper get used.